### PR TITLE
Pull translations from InnerTune Weblate

### PR DESCRIPTION
**Pull request by lucasmz-dev**
InnerTune can be translated here: https://toolate.othing.xyz/projects/innertune
This attempts at pulling some of the work done there

### CODE REVIEW

**Comment by lucasmz-dev** [comment]
I'm unsure about one thing, I think you guys removed strings related to lyrics translation and I'm not sure if I should remove them. (I'm not sure you did)

**Comment by mikooomich** [comment]
Hi. Thanks for your pr! Which lyric translation strings specifically?

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -153,127 +138,116 @@
...
-    <string name="app_version">Versão do aplicativo</string>
-
-    <string name="new_version_available">New version available</string>
-    <string name="translation_models">Translation Models</string>
-    <string name="clear_translation_models">Clear translation models</string>
-</resources>
+    <string name="app_version">Versão do app</string>
+    <string name="new_version_available">Nova versão disponível</string>
+    <string name="translation_models">Modelos de Tradução</string>
+    <string name="clear_translation_models">Limpar modelos de tradução</string>
```
Comment (lucasmz-dev) [inline app/src/main/res/values-pt-rBR/strings.xml]: These two translation model strings, I'm a bit confused to be honest.

When I was attempting to pull stuff, I got some reference to the removal of these but afterwards I didn't and unsure what that was about.

Comment (lucasmz-dev) [reply app/src/main/res/values-pt-rBR/strings.xml]: @mikooomich forgot to mention

Comment (mikooomich) [reply app/src/main/res/values-pt-rBR/strings.xml]: Ah, that can be removed. We don't have the lyric translations libs anymore.

**Comment by lucasmz-dev** [comment]
Removed!

**Comment by lucasmz-dev** [comment]
By the way, I'd like to try this app, but I'm not seeing any prebuilts, do I have to build it atm?

**Comment by mikooomich** [comment]
Eh? You have haven't built it yet? We do require PR authors to validate their changes before submission.

There are no releases as of now.

**Comment by lucasmz-dev** [comment]
No sorry, I'll try to do that.
Translations are less problematic, usually

**Comment by lucasmz-dev** [comment]
Eh let's leave this. 1. I don't know how to build apps :clown_face: (though I suppose your Github Actions setup would've worked)  and 2. it isn't very reliable to be pulling stuff from a repo that is so different here and it'd be hard to maintain.

 I think this might be better done once OuterTune deploys an actual translation solution, and then you might pull for the existing strings to get a bit of a headstart, but there's a bunch of new strings as well compared to the previous upstream. I believe you can actually do this more cleanly anyway using something like Weblate.

If you changed any source strings (or even just, how the string is presented), without removing the translations, their meaning could have been changed and translations can start to get wrong, so that's another thing.

I'm open to maintaining pt-BR if/when this app releases.

**Pull request closed without merging**